[DOC] added singularity build and usage (#28)
tractometry/pyAFQ: docs/source/howto/installation_guide.rst

You can pull the latest of this image or use a specific commit or tag as well::

  docker pull ghcr.io/nrdg/afqsi:latest

> How to build a Singularity image

Reviewer: ```suggestion
How to build a Apptainer (Singularity) image
```

> How to build a Apptainer (Singularity) image

Reviewer: ```suggestion
How to build an Apptainer (Singularity) image
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> If the user intends to execute pyAFQ is a program from the command line (``$pyAFQ /path/to/config.toml``)

Reviewer: Another small typo: 

```suggestion
If the user intends to execute pyAFQ as a program from the command line (``$pyAFQ /path/to/config.toml``)
```

> in an administered environment where root access is not available (e.g. High Performance Computing cluster)

Reviewer: ```suggestion
in an administered environment where root access is not available (e.g., High Performance Computing cluster)
```

> then one solution is to build a Sinularity image from a local pull of the pyAFQ docker container.

Reviewer: ```suggestion
then one solution is to build a Apptainer (also known as Singularity) image from a local pull of the pyAFQ docker container.
```

Start by running a docker registry::

  docker run -d -p 5000:5000 --restart=always --name registry registry:2

Next, create a local tag which allows a push of the pyAFQ container to the local registry::

  docker tag pyAFQ localhost:5000/pyafq
  docker push localhost:5000/pyafq

Finally build the singularity referencing the local registry::

>   SINGULARITY_NOHTTPS=1 singularity build pyafq_latest.simg docker://localhost:5000/pyafq

Reviewer: Does anyone here know if the apptainer call is essentially identical?

Author: I was able to build via: 
`APPTAINER_NOHTTPS=1 apptainer build pyafq_latest.sif docker://localhost:5000/pyafq`

Also, I was imprecise with the contain name on line 94, it would be better to read:
`docker tag ghcr.io/nrdg/pyafq:latest locahost:5000/pyafq`

Reviewer: Do you want to change that here please?

Author: Done: a8312d2d2b44c94692a92594ed90a9b09753318b
